content/2017.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,433 @@
+`nvm <https://github.com/creationix/nvm#install-script>`_.
+
+Once you installed nvm you can use::
+
+    nvm install node
+
+And then you wil be able to run the previous ``npm`` command.
+
+Looking for an IDE?
+-------------------
+
+Don't hesitate to use `Atom <https://atom.io/>`_, it will come with
+``elm-format`` activated each time you save your file which is really
+handy.
+
+Take a few moment to make sure you configured your text editor to
+works friendly with Elm files.
+
+
+Starting your first project
+===========================
+
+No need for a boilerplate here, you can just start by running
+``elm-package install`` and approve the plan:
+
+.. code-block:: bash
+
+    ~/tutorial$ elm-package install
+
+    Some new packages are needed. Here is the upgrade plan.
+
+      Install:
+        elm-lang/core 5.1.1
+        elm-lang/html 2.0.0
+        elm-lang/virtual-dom 2.0.4
+
+    Do you approve of this plan? [Y/n] Y
+    Starting downloads...
+
+      ● elm-lang/html 2.0.0
+      ● elm-lang/virtual-dom 2.0.4
+      ● elm-lang/core 5.1.1
+
+    Packages configured successfully!
+
+You now have a ``elm-package.json`` in your project as well as a
+``elm-stuff/`` directory that contains the library elm-package
+installed.
+
+.. code-block:: bash
+
+    ~/tutorial$ tree -L2
+	.
+    ├── elm-package.json
+    └── elm-stuff
+        ├── exact-dependencies.json
+        └── packages
+
+    2 directories, 2 files
+
+
+Creating your first file
+========================
+
+No need for a boilerplate here, to get started you can simply create a
+new file named ``Main.elm`` with the following:
+
+.. code-block:: elm
+
+    import Html
+    main = Html.text "Hello world"
+
+Playing with elm-format
+-----------------------
+	
+If your editor is well configured with ``elm-format``, you should see
+that as soon as you save it:
+
+.. code-block:: elm
+
+    module Main exposing (..)
+    
+    import Html
+    
+    
+    main =
+        Html.text "Hello world!"
+
+If not, you can run ``elm-format`` manually on your file:
+
+.. code-block:: bash
+
+    ~/tutorial$ elm-format Main.elm
+    This will overwrite the following files to use Elm's preferred style:
+
+        Main.elm
+
+    This cannot be undone! Make sure to back up these files before proceeding.
+
+    Are you sure you want to overwrite these files with formatted versions? (y/n)
+
+    y
+    Processing file Main.elm
+
+To avoid having to answer yes each time you can use the ``--yes`` parameter::
+
+    elm-format --yes Main.elm
+
+
+Opening your app in the browser
+-------------------------------
+
+One way to run your app is to use ``elm-reactor``.
+
+.. code-block:: bash
+
+    ~/tutorial$ elm-reactor
+	elm-reactor 0.18.0
+    Listening on http://localhost:8000
+
+Then open: http://localhost:8000/Main.elm
+
+You are now viewing your Elm app live !!!
+
+
+Learning about the Elm virtual dom
+==================================
+
+The elm virtual dom is exposed in the ``Html`` module.
```

Comment:
`Elm`. Virtual DOM should maybe be capitalized? Considering that you felt a need to explain what a DOM was above, I'm surprised you didn't feel a need to explain the idea of the virtual DOM. Maybe you can just say `Functions to generate HTML are in the HTML module`?